docs/openoni.md

```diff
@@ -0,0 +1,190 @@
+
+restorecon -F -R /var/local/www/django/openoni/
+```
+
+### Python Virtual Environment
+Run these commands as a regular user rather than root
+
+```bash
+cd /var/local/www/django/openoni
+
+# Create and activate Python virtual environment
+virtualenv ENV
+source ENV/bin/activate
+
+# Update pip and setuptools
+pip install -U pip
+pip install -U setuptools
+
+# Install / update OpenONI dependencies
+pip install -U -r requirements.pip
+```
+
+### Migrate Database
+Run these commands as a regular user rather than root
+
+```bash
+cd /var/local/www/django/openoni
+source ENV/bin/activate
+./manage.py migrate
+```
+
+### Newspaper Data Symlink
+Run these commands as a regular user rather than root
+
+```bash
+cd /var/local/www/django/openoni
+source ENV/bin/activate
+
+rm -rf data/batches
+ln -s /var/local/newspapers data/batches
+
+./manage.py batches
+```
+
+
+## Configure
+
+### Solr Schema
+```bash
+cp /var/local/www/django/openoni/docker/solr/schema.xml /var/solr/data/openoni/conf/schema.xml
+cp /var/local/www/django/openoni/docker/solr/solrconfig.xml /var/solr/data/openoni/conf/solrconfig.xml
+chown -R solr.solr /var/solr/data/openoni
+
+service solr restart
+```
+
+### Django
+
+#### Local Settings
+```bash
+cp settings_local_example.py settings_local.py
+```
+
+Follow instructions within for the appropriate deployment environment
+
+#### Theme and Plugins
+[Add theme and plugins](/README.md#including-in-the-open-oni-app)
+
+#### Logging
+Create symlink at `/var/log/openoni` to `/var/local/www/django/openoni/log`
+
+```bash
+ln -s /var/local/www/django/openoni/log /var/log/openoni
+```
+
+#### URLs
+For development, use example file as is:
+```bash
+cp onisite/urls_example.py onisite/urls.py
+```
+
+For use with Nebraska Theme:
+
+`vim onisite/urls.py`:
+```python
+from django.conf.urls import url, include
+
+urlpatterns = [
+  url(r'^calendar-', include("onisite.plugins.calendar.urls")),
+  url(r'^$', include("onisite.plugins.featured_content.urls")),
+  url(r'^map', include("onisite.plugins.map.urls")),
+
+  url('', include("themes.nebraska.urls")),
+  url('', include("core.urls")),
+]
+```
+
+#### WSGI Path
+`vim onisite/wsgi.py`:
+```python
+…
+#sys.path.append('/opt/openoni')
+sys.path.append('/var/local/www/django/openoni')
+…
+```
+
+
+## Compile Static Assets
+Run these commands as a regular user rather than root
+
+```bash
+cd /var/local/www/django/openoni
+source ENV/bin/activate
+
+./manage.py compilescss
+./manage.py collectstatic -c
+
+# Grant write access for both Apache and group
+sudo chown -R apache static/compiled/
+sudo chmod -R g+w static/compiled/
+```
+
+## Load Batches
+Run these commands as a regular user rather than root
+
+```bash
+# Repeat as necessary
+./manage.py load_batch /var/local/www/django/openoni/data/batches/(batch_name)/
+
+# Run a script with nohup in the background to ingest multiple batches quietly
+nohup (command) >> nohup.out
```

Comment:
heckyeah

Reply:
but for real, this is really nice to have in the docs, since I only started using it a few months ago but it would have been awesome to have known about years ago

Reply:
That was the first I'd heard of it too. Hoping this keeps it in mind for future use cases :smile: